src/main/java/org/dave/compactmachines3/command/CommandSchemaLoad.java

```diff
@@ -43,14 +43,14 @@ public void execute(MinecraftServer server, ICommandSender sender, String[] args
         }
 
         Schema schema = SchemaRegistry.instance.getSchema(args[0]);
-        int coords = StructureTools.getCoordsForPos(sender.getCommandSenderEntity().getPosition());
-        EnumMachineSize machineSize = WorldSavedDataMachines.INSTANCE.machineSizes.get(coords);
-        if(machineSize != schema.getSize()) {
-            throw this.getException(sender, "machine_size_does_not_match");
+        int id = WorldSavedDataMachines.getInstance().getMachineIdFromBoxPos(sender.getCommandSenderEntity());
```

Comment:
I understand the intent from context, but `BoxPos` on a `PlayerEntity` instance is a bit misleading here

Reply:
I think what I was going for with `BoxPos` was "a position inside the box", which an entity could be considered a position. If you have a better name for this overloaded method, feel free to tell me.

Reply:
I have updated this method to be called `getMachineIdFromEntityPos` for the entity overload method.